control-plane/config/crd/bases/consul.hashicorp.com_servicedefaults.yaml

```diff
@@ -185,6 +185,63 @@ spec:
                   unlock usage of the service-splitter and service-router config entries
                   for a service.
                 type: string
+              rateLimits:
+                description: RateLimits is rate limiting configuration that is applied
+                  to inbound traffic for a service. Rate limiting is a Consul enterprise
+                  feature.
+                properties:
+                  instanceLevel:
+                    description: InstanceLevelRateLimits represents rate limit configuration
+                      that are applied per service instance.
+                    properties:
+                      requestsMaxBurst:
+                        description: "RequestsMaxBurst is the maximum number of requests
+                          that can be sent in a burst. Should be equal to or greater
+                          than RequestsPerSecond. If unset, defaults to RequestsPerSecond.
+                          \n Internally, this is the maximum size of the token bucket
+                          used for rate limiting."
+                        type: integer
+                      requestsPerSecond:
+                        description: "RequestsPerSecond is the average number of requests
+                          per second that can be made without being throttled. This
+                          field is required if RequestsMaxBurst is set. The allowed
+                          number of requests may exceed RequestsPerSecond up to the
+                          value specified in RequestsMaxBurst. \n Internally, this
+                          is the refill rate of the token bucket used for rate limiting."
+                        type: integer
+                      routes:
+                        description: Routes is a list of rate limits applied to specific
+                          routes. Overrides any top-level configuration.
+                        items:
+                          description: InstanceLevelRouteRateLimits represents rate
+                            limit configuration applied to a route matching one of
+                            PathExact/PathPrefix/PathRegex.
+                          properties:
+                            pathExact:
+                              type: string
+                            pathPrefix:
+                              type: string
+                            pathRegex:
+                              type: string
+                            requestsMaxBurst:
+                              type: integer
+                            requestsPerSecond:
+                              type: integer
+                          required:
+                          - pathExact
+                          - pathPrefix
+                          - pathRegex
+                          - requestsMaxBurst
+                          - requestsPerSecond
```

Comment:
I'm not sure how this was generated and what I need to do about this. Do I edit the "required" fields directly?